include/bluetooth/scan.rst

```diff
@@ -58,45 +75,49 @@ Filters types
+|              | * All specified UUIDs match.                                                                              |
+|              | * At least one filter for each of the filter types set matches.                                           |
+|              |                                                                                                           |
+|              | For example, several filters can be set for name, address, UUID, and appearance.                          |
+|              | To trigger a filter match callback in this scenario, all of the following types must match:               |
+|              |                                                                                                           |
+|              | * At least one of the address filters                                                                     |
+|              | * At least one of the name filters                                                                        |
+|              | * At least one of the appearance filters                                                                  |
+|              | * All of the UUID filters                                                                                 |
+|              |                                                                                                           |
+|              | If not all of these types match, the ``not found`` callback is triggered.                                 |
++--------------+-----------------------------------------------------------------------------------------------------------+
+
+.. _nrf_bt_scan_readme_directedadvertising:
 
 Directed Advertising
 ====================
 
-To support receiving Directed Advertising packets with the Scanning Module, enable one of the following options in Zephyr:
 
-* :option:`CONFIG_BT_PRIVACY` (scanning with changing addresses)
-* :option:`CONFIG_BT_SCAN_WITH_IDENTITY` (scanning with a local identity address)
+To receive directed advertising packets using the Scanning Module, enable one of the following options in Zephyr:
+
+* :option:`CONFIG_BT_PRIVACY` - Scan with changing addresses
+* :option:`CONFIG_BT_SCAN_WITH_IDENTITY` - Scan with a local identity address
+
+It is recommended to enable the :option:`CONFIG_BT_PRIVACY` option to support directed advertising only between bonded peers.
+Use the :option:`CONFIG_BT_SCAN_WITH_IDENTITY` option only when the :option:`CONFIG_BT_PRIVACY` option is not available.
 
-It is recommended to use the privacy option.
-When privacy is enabled directed advertising is only supported from bonded peers.
-Use scanning with identity only when scanning with privacy is not possible.
+When the scanning module is set in advanced mode and :ref:`filters <nrf_bt_scan_readme_filters>` are set, you can use the proprietary ``filter_no_match`` event to check if directed advertising packets have been received.
```

Comment:
Sentence is much clearer now, but please cemove "proprietary", it makes no sense in this context.

Reply:
Thanks for the feedback. Fixed.